How to generate Google API keys.md

```diff
@@ -1,5 +1,7 @@
 # How to generate Google API keys
 
+> Disclaimer: The most recent information can be found at [chrome developers documentation](https://developer.chrome.com/docs/webstore/using_webstore_api/)
```

Comment:
If you change this "disclaimer" to say something like "Step 15 is out of date. Please refer to issue XYZ or look for an alternative in DOCS_LINK. PR welcome to fix it"